### ISSUE TITLE
What is the current status and the plan for this project (Question)

### ISSUE THREAD

**Issue by JavorszkyGaborOrianaWorld** [highcharts/highcharts-editor#215]
Hi,

I would like to ask, what is the current status and the plan for this project, as the latest release is almost 2 years old and the last commit happend more than a year ago.

Thanks!

BR,
Gabor

**Comment by solverat** [comment]
Almost four years later: Any update?

 There are a lot of deprecations if we're using the latest highcharts (v12) with the editor (for example `lang.accessibility`).

**Comment by AidMen** [comment]
While searching through the web I came across the [Highcharts Roadmap (a blog post revised June 30th, 2025)](https://www.highcharts.com/blog/roadmap/) and also reading the blog post about [Introducing Highsoft Lab](https://www.highcharts.com/blog/news/introducing-highsoft-lab/) contains a link pointing to [Highcharts Editor V2 on Highsoft's lab](https://lab.highsoft.com/editorv2/). This lets me get the feeling that something is happening around the Highcharts Editor and I hope that it will be continued.

Is it true @TorsteinHonsi ?

**Comment by TorsteinHonsi** [comment]
@AidMen Yes, we're working on an Editor v2. Currently it is in the early innovation phase, so it's hard to promise anything regarding the timeline.

**Comment by AidMen** [comment]
Does it mean that Highcharts Editor v1 will be discontinued or is there a possibility that it could still be developed?
